app/src/main/java/com/example/drawingApp/DrawingFieldView.kt

```diff
@@ -20,6 +20,14 @@ class DrawingFieldView @JvmOverloads constructor(
     defStyle: Int = 0
 ) : View(context, attrs, defStyle) {
 
+    /**
+     * Whenever a line is being drawn call this
+     * specifically triggers on motion event move
+     * Purpose is to update the model's bitmap whenever we are drawing on the bitmap
+     * to keep the state of the bitmap consistent in the case of a outside interference
+     * such as a phone call or a sudden orientation change
+     */
+    var onLineDrawing: ((Bitmap) -> Unit)? = null
```

Comment:
Let's make this `onBitmapUpdated` because it won't just be lines in the future

Reply:
changed